.github/workflows/compat-check-latest-stable.yml

```
@@ -48,4 +48,3 @@ jobs:
       lightning-version: latest
       pennylane-version: stable
     secrets: inherit
```

To fix the EOF rendering issue:
```suggestion
    secrets: inherit

```